### ISSUE TITLE
False negative for extraneous whitespace around colon

### ISSUE THREAD

**Issue by myint** [PyCQA/pycodestyle#321]
@anthonymayer reported this in hhatto/autopep8#156.

`example.py`:

``` python
x = [1, 2, 3, 4]
x[1: 3]
```

I would expect a message about extraneous whitespace after the colon, but `pep8` seems to report nothing.

```
$ pep8 --version
1.5.7
$ pep8 example.py
```

**Comment by bersbersbers** [comment]
After 3.5 years, would this not be nice to have? :)
